patches/sitecustomize.py

```diff
@@ -73,3 +75,29 @@ def exec_module(self, module):
 
 if not hasattr(sys, 'frozen'):
     sys.meta_path.insert(0, GcpModuleFinder())
+
+@wrapt.when_imported('google.generativeai')
+def post_import_logic(module):
+    if os.getenv('KAGGLE_DISABLE_GOOGLE_GENERATIVE_AI_INTEGRATION') == 1:
+      return
+    if os.getenv('KAGGLE_DATA_PROXY_TOKEN') == 0 || os.getenv('KAGGLE_USER_SECRETS_TOKEN') == 0:
```

Comment:
That's on me, I think I was just blindly following this comment: https://github.com/Kaggle/docker-python/pull/1329#discussion_r1411197882

Reply:
<img width="253" alt="image" src="https://github.com/Kaggle/docker-python/assets/2916088/577f83bf-0203-40b5-bebb-1da39fc1a722">
True is equal to 1 and False is equal to 0 in Python, but I do agree that for legibility I should be checking for True and False directly instead of binary ints